document healthcheck and scaling options for edge applications (#78)
wasmerio/docs.wasmer.io: pages/edge/configuration/app-configuration.mdx

Fields for http healthcheck:
- #### `method`: Required
  HTTP method to use. 
- #### `path`: Required
  Path to send request to. 
- #### `interval`: Required
  Duration between each request
- #### `timeout`: Required
  Timeout duration for requests
- #### `unhealthy_threshold`: Required
  After `unhealthy_threshold` amount of failure of the http requests, app is considered unhealthy and will be restarted
- #### `healthy_threshold`: Required
  After `healthy_threshold` amount of failure of the http requests, app is considered healthy and will be restarted
- #### `expected_status_codes`: Required
  If the application returns these status codes, then the request will be count as successful
- #### `expected_body_includes`:
  If this is specified, response body should include the string defined in this field to be count as successful
- #### `expected_body_regex`:
  If this is specified, response body should be able to match the regular expression defined in this field to be count as successful

So in the example above, if the application fails two consequent request, application will be restarted. It will be keep restarted until it successfully responds to one request

### `scaling`

> The `scaling` field is used to configure number of instances spawned for the application

Reviewer: ```suggestion
The `scaling` field is used to configure how your application scales on Edge.
```

- Required: `false`

Fields for `scaling`:
- #### `mode`: Required
  Possible values : `single_concurrency`

>   In this mode, the application will always have 1 instance

Reviewer: ```suggestion
  In this mode, Edge will only send a single request to each running Webassembly instance at a time. Instances are still reused for multiple requests, but only one after the other. Multiple instances will be started and shut down dynamically, depending on the incoming volume of requests.
  **This is not the preferred deployment mode!**
  It will not scale as well as instances handling multiple requests at once.
  Use this mode i your application does not support handling multiple requests concurrently, like synchronous Python, PHP, ...
```
